src/manager_service.py

```diff
@@ -56,7 +56,7 @@ def setup(state: CharmState, app_name: str, unit_name: str) -> None:
     _setup_service_file(config_file)
     _enable_service()
 
-
+# TODO: Use pipx over pip once the version that supports `pipx install --global` lands on apt.
```

Comment:
Please don't add TODOs to the code for things which will not be tackled immediately. These TODO's generally tend to live forever and confuse other developers.

Reply:
Added